src/controllers/organizations.js

```diff
@@ -0,0 +1,62 @@
+const _ = require('lodash');
+const bcrypt = require('bcrypt');
+const { Organization } = require('../models/user');
+const storage = require('../db/storage');
+const variables = require('../utility/variables');
+const utils = require('../utility/utils');
+const { HASH_ROUNDS } = require('../utility/variables');
+
+async function updateOrgProfile(req, res) {
+    const orga = await Organization.findOne({ _id: req.user.id });
+    if (!orga) {
+        return res.status(403).json({ message: 'User Not Authorised!' });
+    }
+    try {
+        const coverImage = req.file;
+        const newOrga = _.pick(req.body, [
+            'email',
+            'password',
+            'name',
+            'description',
+            'categories',
+            'city',
+            'websiteUrl',
+        ]);
+        if (newOrga.email === orga.email) {
```

Comment:
@sarahmood, you are checking here if the is used by the same orga before, which is actually ok. What I meant before is checking if the new email is used before by other user/orga on the application. You can do this by checking if the BaseUser model has a user with the new email.